### PR TITLE
Use qualified names as IDs for LIonCore/the M3

### PR DESCRIPTION
**Pull request by dslmeinte**

### CODE REVIEW

**Comment by dslmeinte** [comment]
Note: I might change some things here and there, and then force-push.

File: models/lioncore.json [hunk 1]
```diff
@@ -1,661 +1,661 @@
 [
   {
-    "type": "jj0Y6KYlHt3_qyFDFAx4CA",
-    "id": "31_xAp4rmNVj6Sdahgjeuw",
+    "type": "LIonCore_M3_Metamodel",
+    "id": "LIonCore_M3",
     "properties": {
-      "97hsg4Fg3ZitV5FaapRLVg": "LIonCore"
+      "LIonCore_M3_Metamodel_qualifiedName": "LIonCore.M3"
```
Comment (enikao) [inline models/lioncore.json:6]: is the dot on the right-hand side on purpose?

File: models/library-imported-from-ecore.json [hunk 2]
```diff
@@ -19,234 +19,234 @@
...
     "id": "Ei9m_HbdmEYg_EwMLLVZ71ERRBZyXH8GHVyOVia8Sqg",
     "properties": {
-      "EvXCQydomgm9DUb1IwkgRg": "title",
-      "dAp79oxccBi-9j6BrQcZLQ": false,
-      "EE8DYkLRyQzaqyxFqlbOAw": false,
-      "XzYEKJ6QzlZuqKPtMXK_kQ": false
+      "LIonCore_M3_NamespacedEntity_simpleName": "title",
+      "LIonCore_M3_Feature_optional": false,
+      "LIonCore_M3_Feature_derived": false,
+      "LIonCore_M3_Property_disputed": false
```
Comment (enikao) [inline models/library-imported-from-ecore.json:68]: what is `Property.disputed`?

File: schemas/lioncore.serialization.schema.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "LIonCore-serialization",
-  "title": "Serialization format specific to LIonCore",
+  "$id": "LIonCore.M3-serialization",
```
Comment (enikao) [inline schemas/lioncore.serialization.schema.json:3]: dot on purpose (also next line)?

**Review by enikao — APPROVED** [review]

File: models/lioncore.json (hunk 1 above)
Comment (dslmeinte) [reply models/lioncore.json:6]: Yup: I thought the dot was OK for use in qualified name fields, and subsequently qualified names get their dots replaced by underscores to get their IDs.

File: schemas/lioncore.serialization.schema.json (hunk 3 above)
Comment (dslmeinte) [reply schemas/lioncore.serialization.schema.json:3]: See 1st response.

File: models/library-imported-from-ecore.json (hunk 2 above)
Comment (dslmeinte) [reply models/library-imported-from-ecore.json:68]: It indicates for a property whether that property is “disputed” in the sense that
it's not in the agreed-on version of M3/LIonCore, but it's necessary
in order to be able to self-define LIonCore.

**Pull request merged**